src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,2608 @@
+[
+{
```

Comment:
Believe it or not, I started out trying to build up this file by hand, adding a new class whenever I encountered a runtime error. That proved unworkable, and fortunately the agent was easier to use than I had anticipated.